### ISSUE TITLE
Upgrading > 4.0.5 ...

### ISSUE THREAD

**Issue by monowai** [jaredpalmer/razzle#1714]
## ❓Question
When trying to upgrade my app from razzle 4.0.5, my `yarn build` stated producing the error `Can't resolve '../node_modules/babel-preset-razzle/node_modules/@babel/runtime/regenerator'"`. I tracked it down to `razzle-plugin-scss`

I tried adding `@babel/runtime` and other ideas I stumbled across on the web, but in the end the only thing that worked was to add `razzle-typescript-module` to [Package.json](https://github.com/monowai/bc-view/blob/master/package.json) and enable the plugin in in [razzle.config.json](https://github.com/monowai/bc-view/blob/master/razzle.config.js) .  

```
module.exports = {
  plugins: [
    {
      name: "typescript"
    },
```

The [documentation](https://razzlejs.org/plugins/razzle-plugin-typescript) suggests that this plugin should not be required so I'm not sure if this is a documentation issue, bug, or misconfiguration on my part, but it did confuse me for a while.

My build now works fine, with all razzle dependencies @ `^4.1.0`, but something broke along the line after upgrading `razzle-plugin-scss` beyond 4.0.5.

**Comment by fivethreeo** [comment]
Using npm or yarn, all razzle packages are latest?

**Comment by monowai** [comment]
Using `yarn`.  Yes - everything is at latest.

**Comment by fivethreeo** [comment]
This happens from 4.0.5 to 4.0.6?

**Comment by fivethreeo** [comment]
https://github.com/jaredpalmer/razzle/compare/v4.0.5...v4.0.6
https://github.com/jaredpalmer/razzle/compare/v4.0.5...v4.1.0

**Comment by monowai** [comment]
Yes - that's where I recall first observing it.  TBH, i parked it for a while and tried again when I saw 4.1 was out.  I think everything worked until I upgraded `razzle-plugin-scss` to `>= 4.0.6`.  I can hack my build to be absolutely certain.
